js/foam/apps/calc/Calc.js

```diff
@@ -49,6 +49,21 @@ CLASS({
       name: 'CalcName',
       value: 'Calculator',
       translationHint: 'name of application for performing simple calculations'
+    },
```

Comment:
This is how Foam lets us specify i18n strings, these are automatically translated if the language is non-eu